modules/core/src/main/resources/artifact-migrations.v2.conf

```diff
@@ -1404,4 +1404,64 @@ changes = [
     groupIdAfter = dev.scalafreaks
     artifactIdAfter = odin-zio
   },
+  {
+    groupIdBefore = com.bucket4j
+    groupIdAfter = com.bucket4j
```

Comment:
If the `groupId` isn't changing is is enough to just specify `groupIdAfter`.
You will find such examples in the file.

Reply:
nice, I removed the `groupIdBefore`'s